Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.view.html

```diff
@@ -76,7 +76,7 @@ <h4 id="systemUsageHeaderName">{{systemUsageName}}</h4>
                            data-ng-disabled="true"
                            value="{{usage.objectOwnerFullName}}">
                 </div>
-                <div class="form-group">
+                <div class="form-group" ng-if="showDateFields">
```

Comment:
not dataFields --> usagePeriod... datefields is also a bad word when used in danish bit it is what they want... we should however keep our wording closer to the actual meaning of the data